### DISCUSSION TITLE
Add Phillip Hue Bulb Integration to Runelite

### DISCUSSION THREAD

**Discussion by Vaksky** [runelite/runelite#15368] [Ideas]
This plugin used to exist and just needed some minor tweaks to be excellent, but since then the developer quit. 

Can someone republish the plugin? Original code is here: https://github.com/Nick-Anzalone/Hue_Implementation

**Comment by pwatts6060** [comment]
2 people including me have tried. There is some dependency issue that is very difficult to fix. Also this isn't the right place for hub plugin discussion

**Comment by pwatts6060** [comment]
The current attempt at it is here: https://github.com/runelite/plugin-hub/pull/2981
